trade-finance-manager-api/api-tests/v1/deals/deals-submit.api-test.js

```diff
@@ -48,6 +48,12 @@ const createSubmitBody = (mockDeal) => ({
   dealType: mockDeal.dealType,
 });
 
+jest.mock('../../../src/v1/api');
+
+const updateGefFacilitySpy = jest.fn((facilityId, facilityUpdate) => Promise.resolve(
+  { ...facilityUpdate },
+));
+
```

Comment:
Should we move to the same place as other TFM api mocks?:

`src/__mocks__/api.js`

This file is a bit messy. Maybe better not to do this - not sure. The drawback is if updateGefFacility is called somewhere else, the mock will need to be added like you have it, or in api mocks

Reply:
I'd move it in there since it's already repeated several times in your updated tests

Reply:
done